### PR TITLE
Ansible roles 1.x documentation

### PR DESCRIPTION
**Pull request by dLobatog**
To be merged after https://github.com/theforeman/foreman_ansible/pull/22

### CODE REVIEW

File: plugins/foreman_ansible/1.x/index.md [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+
+![registering a host](http://i.imgur.com/mlnVFJj.gif)
+
+If you want to assign Ansible roles to your hosts, please put them on `/etc/ansible/roles/`. After that the plugin will detect them and you will be able to assign them to your hosts when you edit them.
+
+![adding roles](http://i.imgur.com/UyeZIq8.png)
+
+To run the playbook with the roles selected, click on the 'play Ansible roles' button and Foreman will run a playbook on the background with these roles. Logs for this playbook can be found under the regular Foreman log, usually `/var/log/foreman/production.log`
+
+![role play](http://i.imgur.com/eU4RENK.png)
```
Comment (domcleal) [inline plugins/foreman_ansible/1.x/index.md]: It'd be better to keep these in the repository with the page, particularly as they're on HTTP URLs so will cause mixed content warnings.

File: plugins/package_selector.html [hunk 2]
```diff
@@ -0,0 +1,5 @@
+Choose an operating system:
```
Comment (domcleal) [inline plugins/package_selector.html]: Did you mean to add this?

File: plugins/foreman_ansible/index.md [hunk 3]
```diff
@@ -1,7 +1,7 @@
 ---
 layout: plugin_index
 title: Foreman Ansible documentation
-versions: [0.x]
+versions: [0.x, 1.x]
```
Comment (domcleal) [inline plugins/foreman_ansible/index.md]: Nitpick, swap the order so users find the latest docs listed first.

**Comment by dLobatog** [comment]
Updated so that 1.x appears first and images are now hosted by us

**Comment by domcleal** [comment]
Thanks @dLobatog!

**Pull request merged**